forked/0-calico.ps1

```diff
@@ -14,54 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 #>
 
-# Install antrea: CNI Provider
+# Force Kubernetes folder
 mkdir -Force C:/k/
 
-#########################################################################################################
-### kubectl:  is needed by calico installer, but TODO, antrea and calico shouldnt need to do this, and we should just
-### download kubectl as part of the generic windows kubernetes setup...
-#########################################################################################################
-$InstallationFiles = @{
-      "https://dl.k8s.io/release/v1.21.0/bin/windows/amd64/kubectl.exe" = "C:/k/kubectl.exe"
-}
+# Required configuration files symlink
```

Comment:
symlink required to pass default parameters pointing to k\config on calico installation